tools/dockerfile/grpc_scan_build/Dockerfile

```diff
@@ -41,6 +41,19 @@ RUN git clone --recursive https://github.com/grpc/grpc.git
 EXPOSE 8181
 
 CMD \
+  # creating a repository for gRPC submodules for local cloning
+  # this list should be updated when the list of submodules for gRPC changes
+  mkdir grpc-submodules && cd grpc-submodules && \
```

Comment:
Also, the submodules only get fetched once you build the Dockerfile. The built docker image can be cached on our Jenkins workers for months, which also means the reference repo will be months old.